homeassistant/components/intellifire/config_flow.py

```diff
@@ -139,10 +139,10 @@ async def async_step_api_config(
 
                 except (ConnectionError, ClientConnectionError):
                     errors["base"] = "iftapi_connect"
-                    LOGGER.info("ERROR: iftapi_connect")
+                    LOGGER.error("ERROR: iftapi_connect")
```

Comment:
Please explain the error with a statement longer than a single word. Eg "Failed to connect". I'd include the exception in the log too. We don't need to prefix the log message with "ERROR". We're logging at error level, that's enough.